fea-rs/src/compile/compile_ctx.rs

```diff
@@ -1888,6 +1897,37 @@ impl<'a, F: FeatureProvider, V: VariationInfo> CompilationCtx<'a, F, V> {
             }
         } else if item.kind() == Kind::Semi {
             // continue
+        } else if item.kind() == Kind::Comment {
+            assert!(item
+                .token_text()
+                .unwrap_or_default()
+                .contains("# Automatic Code"));
```

Comment:
I don't love the assert, shouldn't we just check if it matches and go about our business if it doesn't? Is it imposible to write some other comment in this position for some reason?